django_auth_ldap/backend.py

```diff
@@ -930,16 +930,16 @@ def _get_group_infos(self):
         return self._group_infos
 
     def _load_cached_attr(self, attr_name):
-        if self.settings.CACHE_GROUPS:
+        if self.settings.CACHE_TIMEOUT != 0:
             key = self._cache_key(attr_name)
             value = cache.get(key)
             setattr(self, attr_name, value)
 
     def _cache_attr(self, attr_name):
-        if self.settings.CACHE_GROUPS:
+        if self.settings.CACHE_TIMEOUT != 0:
```

Comment:
You can drop this guard entirely. `cache.set()` will handle 0.

Reply:
I traced into Django code and it seems there's no special case for `timeout<=0`, at least with `FileBasedCache`, the cached data still gets written out/loaded even though it will never be used. Seemed like a good idea to avoid that overhead, since it's the default too.

Reply:
You make a good point. Ok, lets keep the guard.